src/status_im/infra/events.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.infra.events
+  (:require
+    [utils.re-frame :as rf]))
+
+;; Log error message and call error handle event if any
```

Comment:
Incomplete comment.

Reply:
It is actually complete, but I rephrased it to be more clear